core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -231,6 +233,15 @@ private[spark] class EventLoggingListener(
     }
   }
 
+  private[spark] def redactEvent(
+    event: SparkListenerEnvironmentUpdate): SparkListenerEnvironmentUpdate = {
```

Comment:
Thanks, I'll go through Spark style guide so I don't cause as much trouble next time.

Thanks for reviewing.